ZenPacks/zenoss/Import4/bin/imp4_check.sh

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
```

Comment:
Yes, the script will look at only one input and overriding all intermediate or result files.
The user can redo it multiple times.
However, your comment points out one thing.. what happens if the user uses a different input file!?
Here, the user can rerun the initialize run command and it will clean up the intermediate files on the import4.